src/pages/home/report/ContextMenu/BaseReportActionContextMenu.tsx

```diff
@@ -124,6 +125,10 @@ function BaseReportActionContextMenu({
     const transactionID = ReportActionsUtils.getLinkedTransactionID(reportActionID, reportID);
     const [transaction] = useOnyx(`${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`);
     const [user] = useOnyx(ONYXKEYS.USER);
+    const policyID = ReportUtils.getReport(reportID)?.policyID;
+    const workspaceAccountID = PolicyUtils.getWorkspaceAccountID(policyID ?? '-1');
+    const [cardList = {}] = useOnyx(ONYXKEYS.CARD_LIST);
```

Comment:
I think it might be cleaner to call this something like userCardList to dsitinguish it more clearly from the admin cards